src/config/config.go

```diff
@@ -23,22 +23,28 @@ import (
 	"io/ioutil"
 
 	"github.com/coreos/ignition/third_party/github.com/camlistore/camlistore/pkg/errorutil"
+	"github.com/coreos/ignition/third_party/github.com/coreos/go-semver/semver"
+)
+
+var (
+	MaxVersion = semver.Version{
+		Major:      2,
+		Minor:      0,
+		PreRelease: "dev",
```

Comment:
Good question. The idea was to leave it as "-dev" while it's in this spec-v2 branch (this PR isn't going into master) and then yank that out when it's merged into master. It would be great if there were other mechanisms in place to help ensure I don't fuck it up (because I will). Did you have any ideas? Maybe something in the tag_release script.